grails-app/services/org/broadinstitute/orsp/ConsentService.groovy

```diff
@@ -130,7 +130,7 @@ class ConsentService implements Status {
     def postDataUseLetter(String consentLocation, InputStream inputStream, String fileName) {
         String url = consentLocation + "/dul"
         HttpBuilder http = getMultipartBuilder(url)
-        String fileType = URLConnection.guessContentTypeFromName(fileName)
+        String fileType = MediaType.APPLICATION_OCTET_STREAM_VALUE
```

Comment:
Question - have you tested this with other types, i.e. text, doc, xls, etc? If they all work, then 👍

Reply:
I did that with .txt, .doc, .xls, .csv, etc. They all work!